Tools/CMake/AMReX_Config.cmake

```diff
@@ -146,11 +145,33 @@ function (configure_amrex AMREX_TARGET)
       endif()
    endif()
 
+   # fast math
+   if (AMReX_FASTMATH)
+       # GPU specific backends set in AMReXParallelBackends.cmake
+       if (AMReX_GPU_BACKEND STREQUAL NONE)
+           # See https://cmake.org/cmake/help/v4.1/variable/CMAKE_LANG_COMPILER_ID.html#variable:CMAKE_%3CLANG%3E_COMPILER_ID
+           target_compile_options(${AMREX_TARGET} PUBLIC
+               $<$<CXX_COMPILER_ID:AppleClang,Clang,CrayClang,GNU,IBMClang,IntelLLVM,XLClang>:-ffast-math>
```

Comment:
Hm, maybe we should provide a CMake interface target for the flags, similar as we do for AMReX_FPE. That way, downstream projects can use a pre-compiled AMReX (w/o FM) and turn FM on in the downstream project without mirroring the logic / lookup of flags as done here.